### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by Daniel-at-github** [AtomLinter/linter-flake8#338]
Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.

Code that shows this issue:
~~~ py
def distance(str1, str2):
    if len
~~~

Continuing the code makes the error dissappear
~~~ py
def distance(str1, str2):
    if len(str1) != len(str2):
        raise ValueError
~~~ 

Debug information:
Atom version: 1.11.1
Flake8 version: `2.1.0 (pep8: 1.4.6, mccabe: 0.2.1, pyflakes: 0.8.1) CPython 2.7.6 on Linux`

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by Daniel-at-github** [comment]
Fixed here :hammer: 

Done with virtualenv:
~~~ bash
mkvirtualenv -p $(which python3) atom
pip install --upgrade flake8
which flake8
~~~

put the result of `which flake8` in the setting: Executable path
and OK
